axis_backend/apps/documents/tests/api/test_document_api.py

```
@@ -0,0 +1,559 @@
+        """Test GET /api/documents/latest-versions/ - Get latest versions."""
+        # Create old version
+        old = Document.objects.create(
+            title="Test Document",
+            type=DocumentType.CONTRACT,
+            url="https://example.com/doc-v1.pdf",
+            uploaded_by=self.user,
+            version=1,
+            is_latest=False
+        )
+        # Create latest version
+        latest = Document.objects.create(
```

Ruff (0.14.5)

Local variable `latest` is assigned to but never used

Remove assignment to unused variable `latest`

(F841)